x-pack/test/case_api_integration/spaces_only/config.ts

```diff
@@ -10,6 +10,7 @@ import { createTestConfig } from '../common/config';
 // eslint-disable-next-line import/no-default-export
 export default createTestConfig('spaces_only', {
   disabledPlugins: ['security'],
-  license: 'basic',
-  ssl: true,
+  license: 'trial',
+  ssl: false,
+  testFiles: [require.resolve('./tests/trial')],
```

Comment:
Are we gonna test both basic and trial in this scenario (spaces_only) ?

Reply:
I was thinking just trial for now. I don't think the basic tests will look that different but happy to add more if you think that's important.

Reply:
You are right. No need to add them.